### ISSUE TITLE
Reddell_1992: Processing submission

### ISSUE THREAD

**Issue by ehwenk** [traitecoevo/austraits.build#223]
**Source**: Reddell and Milnes (1992)  /https://doi.org/10.1071/BT9920223
**Custodian**: ?Dataset custodian?
**Handled by**: ?Austraits team member?

- [ ] Contact made with contributor (label as `waiting on response` if waiting for files)
- [ ] Files received from contributor saved in google drive folder `austraits/data submissions/new`  ( label as `ready` )
- [ ] Started importing files into own Austraits branch  (mark label as `In progress` and assign issue to relevant person )
- [ ] Metadata complete, builds into Austraits
- [ ] Report reviewed by AusTraits team
- [ ] Report sent to contributor ( label as `waiting on response` )
- [ ] Response received from contributor
- [ ] Files updated & report finalised
- [ ] Final checks ( all tests pass )
- [ ] Branch merged into master branch (issue `closed` )

NB: 

- if you need to change the config files, e.g. to add a unit conversion or define a new trait. This should be done on the master branch (via pull or rebase)
